### PR TITLE
Change "lightning network" to lowercase

### PR DESCRIPTION
**Pull request by jakubszwedo**
This PR changes "lightning network" to lowercase across the whole guide, except when it's the beginning of the sentence, a title for a section or subsection or when it's a part of a research/paper title.

Related discussion: #890 

👀 [Preview](https://deploy-preview-930--bitcoin-design-site.netlify.app/)

### CODE REVIEW

**Review by Bosch-0 — DISMISSED** [review]
LGTM, thanks a bunch for this Jakub!

File: guide/daily-spending-wallet/sending.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ imagesInfo:
 imagesProcessing:
     - file: processing
       alt: Payment screen showing the transaction is being sent
-      caption: Lightning transactions typically complete in seconds and don't require loaders.
+      caption: lightning transactions typically complete in seconds and don't require loaders.
```
Comment (pavlenex) [inline guide/daily-spending-wallet/sending.md:62]: Well this one is beginning of the sentence?

File: guide/daily-spending-wallet/sending.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ imagesReview:
     - file: enter-pin-before-payment
       alt: Enter PIN screen
       caption: Optionally, this wallet is asking the user to enter their PIN as the final step before paying.
-imagesLightning:
+imageslightning:
```
Comment (GBKS) [inline guide/daily-spending-wallet/sending.md:95]: This is a variable name and should not be changed.

File: guide/daily-spending-wallet/sending.md [hunk 3]
```diff
@@ -307,13 +307,13 @@ Effectively supporting users when problems occur can build trust and confidence,
...
+However, this can be challenging with the variety of different payment formats between on-chain and lightning. What happens when the user is trying to pay somebody, but the receiving party has given them an on-chain address instead of a lightning invoice?
 
-If your wallet is Lightning-only, the user will be unable to proceed with making payment. However, even if your wallet allows the user to send on-chain payments, this payment could still result in a higher transaction fee than they would have incurred over Lightning. If it's in the user's best interest to pay over Lightning, then let them know and help them determine what to do next.
+If your wallet is lightning-only, the user will be unable to proceed with making payment. However, even if your wallet allows the user to send on-chain payments, this payment could still result in a higher transaction fee than they would have incurred over lightning. If it's in the user's best interest to pay over lightning, then let them know and help them determine what to do next.
 
-{% include image-gallery.html pages = page.imagesLightning %}
+{% include image-gallery.html pages = page.imageslightning %}
```
Comment (GBKS) [inline guide/daily-spending-wallet/sending.md:316]: Variable name again. Should stay as is.

File: guide/resources/design-research.md [hunk 4]
```diff
@@ -133,23 +133,23 @@ _2022 by BlockFi._
...
+The State of lightning provides a comprehensive overview of usage on the lightning network seen to date (October 2021). The report highlights and explains the growth in network metrics, provides estimates of mainstream adoption, and looks into the future of the lightning network.
 
 #### [Bitcoin UX Research](https://patestevao.com/work/bitcoin-ux-research/)
 
 _2021 by Estevão._
 
 The goal of the research was to provide an in-depth qualitative knowledge of how people use or perceive bitcoin.
 
-#### [A Deep Dive Into Lightning as a Bitcoin Scaling Solution](https://downloads.coindesk.com/research/Lightning+Network+-+CoinDesk+Research+-+2021.09.pdf)
+#### [A Deep Dive Into Lightning as a Bitcoin Scaling Solution](https://downloads.coindesk.com/research/lightning+Network+-+CoinDesk+Research+-+2021.09.pdf)
```
Comment (GBKS) [inline guide/resources/design-research.md:148]: Used in a link, should not be changed.

File: guide/how-it-works/lightning-services.md [hunk 5]
```diff
@@ -320,7 +320,7 @@ Now, lets learn how [nodes]({{ '/guide/how-it-works/nodes/' | relative_url }}) w
 
 {% include next-previous.html
    previousUrl = "/guide/how-it-works/liquidity/"
-   previousName = "Lightning liquidity"
+   previousName = "lightning liquidity"
```
Comment (GBKS) [inline guide/how-it-works/lightning-services.md:323]: Start of a tooltip sentence, should not be changed.

File: guide/how-it-works/lightning-services.md [hunk 6]
```diff
@@ -239,13 +239,13 @@ To prevent this, ensure your users can [manually backup their channel]({{ '/guid
...
 
-A [Lightning address](https://lightningaddress.com/) LWS service gives users a human-readable address that can be re-used to receive lightning payments. An example is bosch@bitcoin.design.
+A [lightning address](https://lightningaddress.com/) LWS service gives users a human-readable address that can be re-used to receive lightning payments. An example is bosch@bitcoin.design.
 
 {% include picture.html
-   image = "/assets/images/guide/how-it-works/lightning-services/Lightning-address.png"
-   retina = "/assets/images/guide/how-it-works/lightning-services/Lightning-address@2x.png"
-   mobile = "/assets/images/guide/how-it-works/lightning-services/Lightning-address-mobile.png"
-   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/Lightning-address-mobile@2x.png"
+   image = "/assets/images/guide/how-it-works/lightning-services/lightning-address.png"
```
Comment (GBKS) [inline guide/how-it-works/lightning-services.md:245]: Used in file names, should not be changed.

File: guide/how-it-works/lightning-services.md [hunk 7]
```diff
@@ -42,11 +42,11 @@ Some of those issues include:
 
 Lightning services offered by third-parties aim to solve these issues.
 
-This section covers two categories of lightning services: Lightning service providers (LSPs), which focus on onboarding and providing connectivity to lightning users, and [lightning wallet servers]({{ '/guide/how-it-works/lightning-services/#what-are-lightning-wallet-servers' | relative_url }}), which provide additional services not relating to connectivity.
+This section covers two categories of lightning services: lightning service providers (LSPs), which focus on onboarding and providing connectivity to lightning users, and [lightning wallet servers]({{ '/guide/how-it-works/lightning-services/#what-are-lightning-wallet-servers' | relative_url }}), which provide additional services not relating to connectivity.
```
Comment (GBKS) [inline guide/how-it-works/lightning-services.md:45]: Used after a colon, should not be changed.

**Review by GBKS — CHANGES_REQUESTED** [review]
Found several spots where capitalization was incorrectly changed. Please review.

**Review by GBKS — APPROVED** [review]
ACK. Thanks for the extra revision.

**Pull request merged**